test/registration/test_correspondence_rejectors.cpp

```diff
@@ -97,15 +97,13 @@ TEST (CorrespondenceRejectors, CorrespondenceRejectionPoly)
   pcl::transformPointCloud (cloud, target, t, q);
   
   // Noisify the target with a known seed and N(0, 0.005) using deterministic sampling
-  boost::mt19937 rng;
-  rng.seed (1e6);
-  boost::normal_distribution<> nd (0, 0.005);
-  boost::variate_generator<boost::mt19937&, boost::normal_distribution<> > var_nor (rng, nd);
+  std::mt19937 rng(1e6);
+  std::normal_distribution<> nd (0, 0.005);
```

Comment:
Better to just set the result type to `float` here and avoid the static cast below.